test/DynamoCoreTests/UnitsOfMeasureTests.cs

```diff
@@ -726,32 +726,348 @@ public void UnitValues()
             var volume = Volume.FromDouble(1.0, VolumeUnit.CubicFoot);
             Assert.AreEqual(volume.UnitValue, Volume.ToCubicFoot);
         }
+
+
+
+        internal class ViewModelUnitsOfMeasureDynTests : DynamoModelTestBase
```

Comment:
hmm - I'm not sure what this is, did you maybe add this or remove this in your last commits?